messages/config-serialize-function.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+module.exports = function({ key, objectKey }) {
+
+    // special case for parsers
+    const isParser = objectKey === "parser" && key === "parse";
+    const parserMessage = `
+    This typically happens when you're using a custom parser that does not
+provide a "meta" property, which is how ESLint determines the serialized
+representation. Please open an issue with the maintainer of the custom parser
+and share this link:
+
+https://eslint.org/docs/latest/extend/custom-parsers#meta-data-in-custom-parsers
+`.trimStart();
```

Comment:
```suggestion
`.trim();
```

To avoid having two blank lines between this link and "If you still have problems...".